lib/matrix.dart

```diff
@@ -30,13 +30,21 @@ export 'src/database/sqflite_encryption_helper.dart';
 export 'src/event.dart';
 export 'src/presence.dart';
 export 'src/event_status.dart';
-export 'src/voip/call.dart';
-export 'src/voip/group_call.dart';
+export 'src/voip/call_session.dart';
+export 'src/voip/group_call_session.dart';
 export 'src/voip/voip.dart';
-export 'src/voip/voip_content.dart';
-export 'src/voip/conn_tester.dart';
-export 'src/voip/utils.dart';
-export 'src/voip/voip_room_extension.dart';
+export 'src/voip/models/call_events.dart';
+export 'src/voip/models/group_call_events.dart';
+export 'src/voip/models/webrtc_delegate.dart';
+export 'src/voip/models/participant.dart';
+export 'src/voip/models/call_backend.dart';
+export 'src/voip/models/key_provider.dart';
+export 'src/voip/utils/conn_tester.dart';
+export 'src/voip/utils/voip_constants.dart';
+export 'src/voip/utils/rtc_candidate_extension.dart';
+export 'src/voip/utils/famedly_call_extension.dart';
+export 'src/voip/utils/types.dart';
+export 'src/voip/utils/wrapped_media_stream.dart';
```

Comment:
Utils generally implies, that something is used internally to help implementing something. Do we really want to export all the public names in those files?